flow-client/src/test/frontend/ConnectTests.ts

```diff
@@ -396,5 +395,111 @@ describe('ConnectClient', () => {
         await client.call('FooEndpoint', 'fooMethod', {fooParam: 'foo'});
       });
     });
+
+    describe('login', () => {
+      afterEach(() => fetchMock.restore());
+
+      it('on invalid credential', async () => {
```

Comment:
Prefer to continue the convention that test descriptions are naturally readable like "it should do [this] when [that]" or so. So try to start with `it('should ...` or similar. For example in this case I'd maybe write something like:
```suggestion
      it('should return an error on invalid credentials', async () => {
```

Reply:
done